tests/integration/test_flow_integration.py

```diff
@@ -31,7 +31,7 @@ def test_no_batch(arr_in: np.ndarray):
     results_arr = DocumentArray(resp[0].data.docs)
     assert len(results_arr) == 1
     assert results_arr[0].embedding is not None
-    assert results_arr[0].embedding.shape == (1000,)
+    assert results_arr[0].embedding.shape == (1280, )
```

Comment:
why has this changed?

Reply:
Before, this was using an incorrect resnet layer. 1000 is the classification layer.